packages/node/examples/bootstrappedApp/package.json

```diff
@@ -18,7 +18,8 @@
   "homepage": "https://github.com/inrupt/solid-client-authn-js#readme",
   "dependencies": {
     "express": "^4.17.1",
-    "yargs": "^16.1.1"
+    "yargs": "^16.1.1",
+    "@inrupt/solid-client-authn-node": "file:../../dist/"
```

Comment:
Not sure this should point to dist? Normally we just go parent directory?

Reply:
Oh right, I tried a bunch of stuff before realizing I was just off one `..` and didn't fix this.